### PR TITLE
CCN3-1218 - Update review status pages content

### PR DESCRIPTION
**Pull request by cruddasj**
## Description

Updates to status pages when an application has been submitted or is undergoing ARC review.

## Todo's before PR

- [X] Branch has been rebased against develop
- [x] Unit tests passed (`make test`)
- [X] PR named appropriately - see [guide](https://github.com/InformedSolutions/OFS-MORE-DevOps-Tooling/wiki/Commits-guideline)
- [X] PR description describes the overall goals of PR commits
- [X] Templates have been checked against the relevant user-story

## Migrations

- [X] Null fields in models specifies default value
- [X] You generated and applied migrations (`make migrate`)
- [X] You updated fixtures (`make export`)

## PR Todo's

Please refer to PR [guide](https://github.com/InformedSolutions/OFS-MORE-DevOps-Tooling/wiki/Pull-requests#how-to-submit-a-pull-request)

- [x] Specified reviewers (even if it's yourself)
- [X] Specified assignees (those who'll merge)
- [X] Specified labels

## PR review

Please refer to PR review [guide](https://github.com/InformedSolutions/OFS-MORE-DevOps-Tooling/wiki/Pull-requests#how-to-review-a-pull_request)

- [x] Code syntax formatting checked
- [x] Debug messages are absent

## PR merge

Select only one:

- [ ] Should be merged?
- [X] Should be rebased?
- [ ] Should be squashed?

Please refer to PR merge [guide](https://github.com/InformedSolutions/OFS-MORE-DevOps-Tooling/wiki/Pull-requests#how-to-merge-a-pull_request)

### CODE REVIEW

File: application/templates/awaiting-review.html [hunk 1]
```diff
@@ -33,7 +40,7 @@ <h2 class="form-title heading-medium">
 <ul class="list list-bullet">
     <li>your name</li>
     <li>the childcare address</li>
-    <li>childcare hours</li>
+    <li>overnight care</li>
```
Comment (ricaduchateau) [inline application/templates/awaiting-review.html]: This requires a full stop at the end, as per the content document.

File: application/templates/application-accepted.html [hunk 2]
```diff
@@ -22,15 +22,21 @@ <h2 class="form-title heading-medium">
...
 <p>We are currently doing background checks on you and people in your home.</p>
 
-<p>This means we are:</p>
+<p>This includes:</p>
 
 <ul class="list list-bullet">
     <li>contacting your references</li>
-    <li>carrying out checks with social services</li>
+    <li>carrying out checks with council social services</li>
     <li>checking your health declaration booklet</li>
```
Comment (ricaduchateau) [inline application/templates/application-accepted.html]: This line requires a full stop at the end, as per the content document.

File: application/templates/application-accepted.html [hunk 3]
```diff
@@ -43,7 +49,7 @@ <h2 class="form-title heading-medium">
 <ul class="list list-bullet">
     <li>your name</li>
     <li>the childcare address</li>
-    <li>childcare hours</li>
+    <li>overnight care</li>
```
Comment (ricaduchateau) [inline application/templates/application-accepted.html]: This requires a full stop at the end, as per the content document.

**Comment by cruddasj** [comment]
@ricaduchateauIS - I'm not sure the content itself is quite correct here in keeping with GOV.UK copy guidance (have dropped a note against relevant Jira tickets)

**Comment by ricaduchateauIS** [comment]
Hi @cruddasj,

I think there are cases of full stops after bullet point text on other pages. I'll look to change these pages as well if Ofsted agrees to drop the full stops.

With kind regards,

Rica

**Pull request merged**